pkg/agent/flowexporter/connections/conntrack_windows.go

```diff
@@ -20,9 +20,9 @@ import (
 	"net"
 
 	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/ovs/ovsctl"
 )
 
-// TODO: Support FlowExporter feature for windows. We have to pass ovsctlClient when supported.
-func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet) *connTrackOvsCtl {
-	return NewConnTrackOvsAppCtl(nodeConfig, serviceCIDR, nil)
+func NewConnTrackSystem(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet, ovsctlClient ovsctl.OVSCtlClient) *connTrackOvsCtl {
+	return NewConnTrackOvsAppCtl(nodeConfig, serviceCIDR, ovsctlClient)
```

Comment:
We do not need ovsCtltClient argument in NewConnTrackOvsAppCtl. This is not part of your PR, but code can be simplified by creating new ovsCtlClient in the following way:
`ovsctlClient := ovsctl.NewClient(connTrackOvsCtl.nodeConfig.OVSBridge)`

We do not have to pass `agentQuerier.GetOVSCtlClient()` in agent.go. You could consider this as an enhancement to simplify the code.

Reply:
Thanks! I have simplified the code in this way.